horovod/_keras/__init__.py

```diff
@@ -13,10 +13,15 @@
 # limitations under the License.
 # ==============================================================================
 
+from distutils.version import LooseVersion
+
 import horovod.tensorflow as hvd
 import tensorflow as tf
 
 
+_PRE_TF_2_4_0 = LooseVersion(tf.__version__) < LooseVersion('2.4.0')
```

Comment:
Why do you create a shortcut ? You use it once.

Reply:
These LooseVersion checks tend to be pretty expensive, as they need to do string manipulation etc.  When this function gets called repeatedly (for example, eager execution), then it's better to call this only once.